packages/perseus-editor/src/widgets/label-image/behavior.tsx

```diff
@@ -88,6 +119,10 @@ const styles = StyleSheet.create({
 
         color: gray17,
     },
+
+    selectLabel: {
+        marginRight: 8,
+    },
```

Comment:
The dropdown for selecting the direction was right up against the description of the box, so added some space. Might update this to a semantic token (`sizing.size_080`).